x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/CsvSpecTestCase.java

```diff
@@ -42,14 +43,8 @@ public CsvSpecTestCase(String fileName, String groupName, String testName, Integ
     protected final void doTest() throws Throwable {
         // Run the time tests always in UTC
         // TODO: https://github.com/elastic/elasticsearch/issues/40779
-        if ("time".equals(groupName)) {
-            try (Connection csv = csvConnection(testCase); Connection es = esJdbc(connectionProperties())) {
-                executeAndAssert(csv, es);
-            }
-        } else {
-            try (Connection csv = csvConnection(testCase); Connection es = esJdbc()) {
-                executeAndAssert(csv, es);
-            }
+        try (Connection csv = csvConnection(testCase); Connection es = esJdbc()) {
```

Comment:
@matriv The previous hack was unnecessary - the conditional should be addressed in `connectionProperties` and further more, `esJdbc()` actually calls `esJdbc(connectionProperties())` so the code was not only duplicated but also identical in results.

Reply:
I don't get how this is addressed only for a specific file, in this case: `time.csv-spec`.

Reply:
It is not but then again, it wasn't before.
In other words, the if ("time".equals(groupName)) does the right thing but calling esJdbc(connectionProperties()) is the same as calling esJdbc() meaning the if and else branch end up calling the same code path.

Reply:
It's not the case, if you check again there are 2 different `connectionProperties()` involved there. and the one for the `time` sets the timezone to UTC where the other uses a random timezone.

Reply:
@costin Did you check this ^^ ?